offchain/rollups-events/src/broker/mod.rs

```diff
@@ -14,14 +14,21 @@ use redis::{
 };
 use serde::{de::DeserializeOwned, Serialize};
 use snafu::{ResultExt, Snafu};
+use std::collections::HashMap;
+use std::convert::identity;
 use std::fmt;
+use std::str::FromStr;
 use std::time::Duration;
 
 pub use redacted::{RedactedUrl, Url};
 
+use crate::Address;
+
 pub mod indexer;
 
 pub const INITIAL_ID: &str = "0";
+const DAPPS_KEY: &str = "experimental-dapp-addresses-config";
+const DAPPS_DIVIDER: &str = ", ";
```

Comment:
Instead of relying on a `string` key with a CSV string, why don't use a `set` ?

This way, whoever wants to add a new application to the claimer to watch, can just call a `SADD` instead of mess with a CSV.

- https://redis.io/docs/latest/develop/data-types/sets/
- https://redis.io/docs/latest/commands/sadd/
- https://redis.io/docs/latest/commands/srem/